### PR TITLE
added arrow dowm svg to dropdowns fixes #47

### LINKED ISSUE #47: Upload page: add dropdown svg's
to dropdown input fields
and
- style drop down menus

### PR DESCRIPTION
**Pull request by nogainbar**
Also added max-width to the whole of form container so that it doesnt stretch of larger screens

### CODE REVIEW

**Review by jbesraa — APPROVED** [review]

**Review by jbesraa — CHANGES_REQUESTED** [review]
please fix merge conflicts

File: public/style.css [hunk 1]
```diff
@@ -21,6 +21,10 @@
   height: 50px;
 }
 
+.form_container {
+  max-width: 330px;
```
Comment (des-des) [inline public/style.css]: Yeah guessing this is for website on desktop, we can use tachyons as this has media queries if we want

**Review by des-des — CHANGES_REQUESTED** [review]

**Comment by des-des** [comment]
@esraajb okay I am gonna focus on getting the email stuff working, do you want to make that change?

**Review by jbesraa — APPROVED** [review]

**Comment by jbesraa** [comment]
@des-des

**Review by des-des — APPROVED** [review]

**Pull request merged**